### ISSUE TITLE
Can't die on servers

### ISSUE THREAD

**Issue by zetcipher** [eidehua/More-Health-Enhanced#5]
Mod Version 6.3
Forge Version 1448

Any damage I receive gets healed instantly, I can't even use the kill command.
Nothing unusual appears in the log. I have not tested this in single player

**Comment by eidehua** [comment]
Could you test it on single player as well? I have tested on my own local server and in sp and I do correctly take damage. Maybe there might be a conflict with another mod?

**Comment by x2835615** [comment]
i think conflict with GraveStone and openblock ..
I think this issue raised in the grave features If you use both
and this issue  doesn't appear to ssp,only smp
i try turn off openblock's grave,But the problem still occurs..
Maybe you can try to repair..
if need more Information,I can provide.
Thanks to the author and apologize for my poor English

**Comment by eidehua** [comment]
Yes please provide more info. Are you using the latest beta build available on the minecraft forums thread?

**Comment by x2835615** [comment]
yes,i try to beta build,but it still occur.
by the way,if damage once deadly,ex:Creeper、Height falls,it will be death
thanks.

**Comment by LemADEC** [comment]
Are you using Cauldron for your server? Can you reproduce it with a plain Forge server?

**Comment by MudkipNinja** [comment]
I know this is kind of an old thread, but perhaps I can expand. I'm having the same issue as @Numba90 and @x2835615. Whenever I run this mod on SMP, acquire more health, then try to take damage, it registers me being hit, but I do not lose any health. On SSP it works fine. I Have several other mods running but as soon as I take the More Health Enchanted mod out, everything works fine. I'm running a standard forge 1.7.10 server, and I've tried both the stable build and beta build of the mod, but the problem still persists. If it's an issue with another mod then I'll try to change some properties, but if anyone could help, that would be great.

**Comment by eidehua** [comment]
Is it possible for you to test this on an SMP server with just forge and my mod? And using my latest beta version of course.

Could you test: 
Dimension changing
Login/logout saving hearts (aka gain hearts, then logout asap)
Server shutdown preserving player's hearts (aka gain hearts, then shutdown server asap).
Different damage sources: monsters, falling, fire/lava 
What difficulty mode you are on.

Please also check the stats screen, and see if the information provided there matches your health information. 

Also can you comment on your specific forge version?

Just want to be thorough with this.

**Comment by MudkipNinja** [comment]
No problem! So, with just your mod on my server and on my client, tried changing dimensions to the nether and end, but the problem still persisted; I couldn't lose health. I Then spawned myself a heart container, used it, then when i got the message in chat saying I gained a new heart, I quickly logged out and logged back in, but it still only looks as though I have 10 hearts in the health HUD, and i still cant lose health. Shutdown the server on heart gain, same issues. I tried fall damage, monsters, wither effect, hunger, lava, fire and drowning, and my health did not visibly go down from any of these, (I was able to just swim around in lava for more than 2 minutes before I got out unscathed.) All of this was tested on easy, normal, hard, and peaceful. I'm using forge build 10.13.4.1448. In the end it would seem that even when I use heart containers, my health doesn't increase past 10, and no matter what health I _should_ have, It never goes above or below 10. Also another peculiar thing I noticed, was that when I pressed "H" to open the health window, it just crashed me game with this: "java.lang.NullPointerException: Rendering screen" as the main error. Any more tests you need me to run, I'll gladly do, its such a cool mod when it works!

**Comment by MudkipNinja** [comment]
@eidehua Anything?

**Comment by eidehua** [comment]
Sorry been busy with my internship, haven't had much time for this!

 Can you list the other mods you are using just in case? I will take a look at the SMP code. I have done some testing myself with my own server and just forge + my mod and it seems to be working so I'm not sure if there is a corner case issue here.

One thing to try: can you update the config file with new settings to starting health/max health? And I'm pretty sure the "H" issue was fixed a few builds ago, not sure why that is a problem still. 

Also can you paste the full config file in a comment, and another comment with the server log? (Start up, and gain some hearts, and take some damage, then log out)
And also the crash error for "H"

Thanks!

**Comment by BLACKFLAGX** [comment]
maybe this reply is too later, i face this issue as well. but i can confrim that it is an issue for this mod, because i found it is Enchantments  cause this issue, so you can close the Armor Enchantments to avoid this issue.

I:"Armor Enchantments ID"=120
B:Enchantments=**false**
